index.ts

```diff
@@ -31,6 +34,12 @@ async function captureHeartbeat(timestamp: string) {
     await posthog.capture('heartbeat', { $timestamp: timestamp })
 }
 
+/** Capture `heartbeat` event, directly into the queue, but with a random distinct ID so that it goes via buffer */
+async function captureHeartbeatBuffer(timestamp: string) {
+    const randomDistinctId = `PostHog Heartbeat Plugin / Buffer ${Math.random().toString().slice(2)}`
```

Comment:
We can end up having collisions here even though it's unlikely. Let's use timestamp in utc with second precision instead (since we're sending these once per minute that will work well). Plus it will be a more helpful name for us too :)

Reply:
That's a good idea 👍